tzdata/Makefile

```diff
@@ -64,131 +43,42 @@ icunew/%/zoneinfo64.txt: mirror/tzdata%.tar.gz mirror/tzcode%.tar.gz
 	@echo "## Rebuilding icu data in $(ICU_SRC)/data"
 	@$(MAKE) -C $(ICU_SRC)/data  2>> $(shell basename $(@D)).log >> $(shell basename $(@D)).log
 
-	@echo "## Testing icu data against tz algorithm"
-	@$(MAKE) -C $(TZSRC) check-dump  2>> $(shell basename $(@D)).log >> $(shell basename $(@D)).log
-	@echo "## Test passed"
+#	@echo "## Testing icu data against tz algorithm"
```

Comment:
Commented out on purpose?

Reply:
Thanks for catching this.. not on purpose. Will fix it soon.

Reply:
Uncommented and run the build again. The test passed without any errors. Committed the change and squashed.

I commented out these lines while debugging the make file because it takes extra time and forgot to uncomment.